sklearn/datasets/tests/test_kddcup99.py

```diff
@@ -38,6 +40,10 @@ def test_percent10():
     assert_equal(data.data.shape, (9571, 3))
     assert_equal(data.target.shape, (9571,))
 
+    X_y_tuple = fetch_kddcup99('smtp', return_X_y=True)
+    bunch = fetch_kddcup99('smtp')
```

Comment:
Don't we already have a bunch above? I think it's unclear to the reader why we get this again

Reply:
Thanks - you're right.   Fixed in https://github.com/scikit-learn/scikit-learn/pull/10774/commits/e68f3f99076b6de01354b5c9ef920075fa7a040f